openhtf/io/frontend/__main__.py

```diff
@@ -244,8 +236,8 @@ def initialize(self, store):
 
   def post(self, host, port, prompt_id):
     msg = json.JSONEncoder().encode(
-      {'id': prompt_id, 'response': self.request.body})
-    self.store.MessageFramework(host, port, method='POST', message=msg)
+        {'id': prompt_id, 'response': self.request.body})
+    self.store.stations[(host, int(port))].Notify(msg)
```

Comment:
i don't think you need the `()` inside the `[]`, do you?  i think you can just do `.stations[host, int(port)].Notify`